packages/@aws-cdk/aws-ecs/lib/load-balanced-fargate-service.ts

```diff
@@ -101,6 +102,12 @@ export interface LoadBalancedFargateServiceProps {
    * Setting this option will set the load balancer port to 443.
    */
   certificate?: CertificateRef;
+  /**
+   * Allows opting out of creation of aws log drivers
```

Comment:
I would just write "whether to add AWS log drivers"

Reply:
done